pkg/validate/sources.go

```diff
@@ -1,48 +0,0 @@
-// Copyright The Shipwright Contributors
```

Comment:
I tend to retain the validation to validate that a type is present and that if for example type is set to Git, that the Git object is also there, but OCIArtefact is `nil`.